packages/react-components/src/molecules/__tests__/UserProfilePersonalText.test.tsx

```diff
@@ -29,7 +29,7 @@ it.each`
 
 it('does not show the location icon if no location is available', () => {
   const { queryByTitle } = render(
-    <UserProfilePersonalText teams={[]} role={'Grantee'} />,
+    <UserProfilePersonalText teams={[]} role={'Grantee'} labs={[]} />,
```

Comment:
Rather than modify every test each time there is a new prop can you do something like https://github.com/yldio/asap-hub/pull/1015/files#diff-c5e608943b86b21f188f1b72070124978e7cd87f080765ce67cde9ce174f5243R7

Reply:
is the link correct? @kmaid

Reply:
No, but I have spoken with @ana-yldio so it already changed